Makefile

```diff
@@ -0,0 +1,31 @@
+# Make GNU Make xonshy
+SHELL=xonsh
+.SHELLFLAGS=-c
+.ONESHELL:
+.SILENT:
```

Comment:
GNU Make has some gems. I figured a xonshified Makefile would be greatly appreciated.